modules/fedimint-lnv2-server/src/lib.rs

```diff
@@ -334,7 +334,7 @@ impl ServerModule for Lightning {
         _dbtx: &mut DatabaseTransaction<'_>,
     ) -> Vec<LightningConsensusItem> {
         let mut items = vec![LightningConsensusItem::UnixTimeVote(
-            duration_since_epoch().as_secs(),
+            60 * (duration_since_epoch().as_secs() / 60),
```

Comment:
nit: let's leave a comment so future generations of developers will know what's up here

Reply:
BTW. It occurred to me that if  you offset each `duration_since_epoch` for each peer by the `(peer_id * interval / peer_num)`, you can recover the timestamp smoothness and compensate for the lag of rounding-down.

Reply:
I feel like that would be over-engineering it. Feel free to add later, with this approach we can always change if we feel we need to.